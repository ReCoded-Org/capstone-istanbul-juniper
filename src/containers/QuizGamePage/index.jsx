import React from "react";
import QuizGame from "../../components/QuizGame";

const QuizGamePage = () => {
  return (
    <>
      <QuizGame />
    </>
  );
};

export default QuizGamePage;
