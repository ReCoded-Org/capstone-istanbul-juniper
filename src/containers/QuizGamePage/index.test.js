import React from "react";
import QuizGamePage from "./index.jsx";
import renderer from "react-test-renderer";

describe("Quiz game page Container", () => {
  it("matches the tree", () => {
    const tree = renderer.create(<QuizGamePage />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
