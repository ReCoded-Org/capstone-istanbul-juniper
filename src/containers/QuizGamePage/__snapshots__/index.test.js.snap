// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Quiz game page Container matches the tree 1`] = `
<div
  className="quizWrapper"
>
  <h1
    className="quizWrapper__header"
  >
    Quiz Game
  </h1>
  <button
    className="quizWrapper__startGame"
    onClick={[Function]}
  >
    Start
  </button>
</div>
`;
